thorlcr/activities/indexread/thindexreadslave.cpp

```diff
@@ -719,10 +710,70 @@ class CIndexReadSlaveBase : public CSlaveActivity
     virtual void gatherActiveStats(CRuntimeStatisticCollection &activeStats) const
     {
         PARENT::gatherActiveStats(activeStats);
+        // *) jhtree stats will have been tracked as follows
+        //    - For superfiles, there will be 1 IContextLogger in contextLoggers[] for each part
+        //    - In all other cases, use 1st IContextLogger in contextLoggers[] for all parts
+        // *) File IO stats will have been tracked as follows
+        //    - the io stats will be tracked in keyManagers[]
+        //    - (There will be exactly 1 keyManager per part)
+        // Note: the KeyManagers[], contextLoggers[] and fileStats[] are not necessarilly the same size
+        // - KeyManagers[] will always be the same size as partDescs[]
```

Comment:
This comment doesn't seem to be quite true, since line 742 is testing it is true.  Worth adding a comment where when it might not be true.